### PR TITLE
lsof: edit -iTCP example with -i6TCP, -n and -P

### PR DESCRIPTION
**Pull request by jerryc05**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/lsof.md [hunk 1]
```diff
@@ -32,6 +32,6 @@
 
 `lsof +D {{path/to/directory}}`
 
-- Find the process that is listening on a local TCP port:
+- Find the process that is listening on a local IPv4 TCP port, don't convert network numbers or port numbers:
 
-`lsof -iTCP:{{port}} -sTCP:LISTEN`
+`lsof -i4TCP:{{port}} -sTCP:LISTEN -n -P`
```
Comment (sbrl) [inline pages/common/lsof.md:37]: Not really a fan of explicitly specifying IPv4 here. IPv4 will be going away eventually - we should be promoting the use and uptake of IPv6 instead.

Comment (jerryc05) [reply pages/common/lsof.md:37]: True, but my intention was to inform the users of the -4 (or -6) flag here. 
I believe the users will realize that they can change -4 to -6 for IPv6.

Comment (sbrl) [reply pages/common/lsof.md:37]: True, but could we specify `-6` instead of `-4` here please?

Comment (jerryc05) [reply pages/common/lsof.md]: Done.

File: pages/common/lsof.md [hunk 2]
```diff
@@ -32,6 +32,6 @@
 
 `lsof +D {{path/to/directory}}`
 
-- Find the process that is listening on a local TCP port:
+- Find the process that is listening on a local IPv6 TCP port, don't convert network numbers or port numbers:
```
Comment (marchersimon) [inline pages/common/lsof.md:35]: I think that reads a bit better:
```suggestion
- Find the process that is listening on a local IPv6 TCP port and don't convert network or port numbers:
```

**Review by marchersimon — APPROVED** [review]
Thanks @jerryc05 
I just left a small suggestion

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @jerryc05!

**Pull request merged**